ComputerVision/SDK/readme.md

```diff
@@ -0,0 +1 @@
+You'll need to have Azure subscription keys to use the SDK.
```

Comment:
Here you will find a set of examples for various Computer Vision scenarios that are written based on the Computer Vision SDK. Just update the code to enter your API Key, select the desired project as the starting project and hit F5 in Visual Studio.